pkg/asset/cluster/tfvars.go

```diff
@@ -239,3 +245,39 @@ func (t *TerraformVariables) Load(f asset.FileFetcher) (found bool, err error) {
 
 	return true, nil
 }
+
+// injectInstallInfo adds information about the installer and its invoker as a
+// ConfigMap to the provided bootstrap Ignition config.
+func injectInstallInfo(bootstrap []byte) string {
+	config := &igntypes.Config{}
+	if err := json.Unmarshal(bootstrap, &config); err != nil {
+		panic("installer bug: failed to unmarshal bootstrap Ignition config")
```

Comment:
:scream: error and bail is also acceptable.

For future when we have ability to return error types, this can be `This is proabably an installer bug` kind :P